### PR TITLE
Updated torch package versions 

### PR DESCRIPTION
**Pull request by harshbafna**
## Description

 - Update torch package versions in the requirements file
 - added support for Cuda 10.1 and 10.2 specific binary
 - Fixes sanity suite for a scenario where `nvidia-smi` command is available but there is no GPU hardware on the instance.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Feature/Issue validation/testing

TBD once torch binaries are publicly available

## Checklist:

- [ ] Have you added tests that prove your fix is effective or that this feature works?
- [ ] New and existing unit tests pass locally with these changes?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [ ] Have you made corresponding changes to the documentation?

### CODE REVIEW

**Comment by chauhang** [comment]
@harshbafna Why do we need so many different requirementsxxx.txt files? Can we not simplify the steps by defining an CUDA_VERSION environment variable and just use that in the requirements.txt

File: frontend/modelarchive/src/test/resources/models/noop_no_archive/noop/service.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
+
+
+"""
+NoopService defines a no operational model handler.
```
Comment (chauhang) [inline frontend/modelarchive/src/test/resources/models/noop_no_archive/noop/service.py:4]: @harshbafna Why are these other files included in this PR? It should only contain torch package version related items

File: frontend/server/src/test/java/org/pytorch/serve/ModelServerTest.java [hunk 2]
```diff
@@ -351,6 +351,33 @@ public void testNoopPrediction() throws InterruptedException {
         testPredictions("noop", "OK", null);
     }
 
+    @Test(
+            alwaysRun = true,
+            dependsOnMethods = {"testSetDefaultVersionNoop"})
+    public void testLoadModelWithNakedDirModelArchive() throws InterruptedException {
```
Comment (chauhang) [inline frontend/server/src/test/java/org/pytorch/serve/ModelServerTest.java:357]: @harshbafna Why are these other files included in this PR? It should only contain torch package version related items. Please split into separate PRs if you are adding additional tests for NakedMAR

File: README.md [hunk 3]
```diff
@@ -39,16 +39,22 @@ TorchServe is a flexible and easy to use tool for serving PyTorch models.
 
 2. Install python pre-requisite packages
 
- - For CPU or GPU-Cuda 10.2
+ - For CPU or GPU with Cuda 11.0
```
Comment (chauhang) [inline README.md:42]: @harshbafna  PyTorch 1.7 also supports cuda 11.0, please add relevant steps for it as well

**Review by chauhang — CHANGES_REQUESTED** [review]
Please see the comments inline

**Comment by harshbafna** [comment]
@chauhan: We will be releasing TorchServe, 0.2.1 from the `patch_release_0_2_1` branch instead of the master branch as it master branch also has changes related to windows native, response envelope, etc.

We created a separate PR forked from the `patch_release_0_2_1` branch. Could you please review #791 instead?

Closing this PR as this is no more a valid PR.
